nodes/clusters/nodes-cluster-enabling-features.adoc

```diff
@@ -28,7 +28,9 @@ For more information about the features activated by the `TechPreviewNoUpgrade`
 
 ** xref:../../storage/container_storage_interface/persistent-storage-csi-sc-manage.adoc#persistent-storage-csi-sc-manage[Managing the default storage class]
 
-** xref:../../authentication/understanding-and-managing-pod-security-admission.adoc#understanding-and-managing-pod-security-admission[Pod security admission enforcement].
+** xref:../../authentication/understanding-and-managing-pod-security-admission.adoc#understanding-and-managing-pod-security-admission[Pod security admission enforcement]
+
+** xref:../../installing/installing_gcp/installation-config-parameters-gcp.adoc#installation-configuration-parameters-additional-gcp_installation-config-parameters-gcp[Installation configuration parameters for GCP]
```

Comment:
Is it for TP feature only?

Reply:
Yes, so since this feature should be GA upon initial release, even if it's in 4.18.1, this link shouldn't be necessary. @sheriff-rh, please watch out for this when you update the feature gate docs ahead of GA.